packages/Datepicker/src/Datepicker.tsx

```diff
@@ -182,18 +195,25 @@ const Datepicker: React.FunctionComponent<DatepickerProps> = ({
     };
 
     const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
-        const { value: inputValue } = e.target;
-        if (inputValue) {
-            const date = DateTime.fromISO(inputValue);
-            if (date.isValid) {
-                if (isDateSelectable(date)) {
-                    handleChange(date);
+        const { value: inputValue } = e.target
+
+        const dateRegEx = /^(?:\d{4}[-/]\d{2}[-/]\d{2}|\d{2}[-/]\d{2}[-/]\d{4})$/;
```

Comment:
This could be a constant outside the component, so it doesn't get initiatize on every input change